lighthouse-core/audits/byte-efficiency/unused-css-rules.js

```diff
@@ -32,7 +29,7 @@ class UnusedCSSRules extends Audit {
     return {
       category: 'CSS',
       name: 'unused-css-rules',
-      description: 'Uses 90% of its CSS rules',
+      description: 'Avoids unnecessary CSS',
```

Comment:
"unnecessary CSS" might not be descriptive enough.
What do you think about Avoids including unnecessary CSS or Avoids loading unnecessary CSS to correlate with the theme of reducing bytes on page load?